schemas/scanorganic-v1.0.json

```diff
@@ -0,0 +1,111 @@
+                    "type"          : "string",
+                    "format"        : "date-time"
+                },
+                "event" : {
+                    "enum"          : [ "ScanOrganic" ]
+                },
+                "horizons": {
+                    "type"          : "boolean",
+                    "description"   : "Boolean value copied from the Journal LoadGame event, when it is present there."
+                },
+                "odyssey": {
+                    "type"          : "boolean",
+                    "description"   : "Boolean value copied from the Journal LoadGame event, when it is present there."
+                },
+                "StarPos": {
+                    "type"          : "array",
+                    "items"         : { "type": "number" },
+                    "minItems"      : 3,
+                    "maxItems"      : 3,
+                    "description"   : "Must be added by the sender"
+                },
+                "ScanType": {
+                    "enum"          : ["Log", "Sample"],
+                    "description"   : "Scan level of the scan; can be 'Log', 'Sample', or 'Analyse'. 'Analyse' can give bad location data and is excluded."
+                },
+                "Genus": {
+                    "type"          : "string",
+                    "minLength"     : 1,
+                    "description"   : "Internal ID of bio genus string."
+                },
+                "Species": {
+                    "type"          : "string",
+                    "minLength"     : 1,
+                    "description"   : "Internal ID of bio species string."
+                },
+                "Variant": {
+                    "type"          : "string",
+                    "minLength"     : 1,
+                    "description"   : "Internal ID of bio variant string. This mirrors the CodexEntry name."
+                },
+                "SystemAddress": {
+                    "type"          : "integer"
+                },
+                "Body": {
+                    "type"          : "integer",
+                    "renamed"       : "BodyID",
+                    "description"   : "Body ID is reported as 'Body' in this event. Rename to BodyID for general cross-compatibility."
+                },
+                "BodyID": {
+                    "type"          : "integer"
+                },
+                "BodyName": {
+                    "type"          : "string",
+                    "description"   : "Must be added by the sender to corroborate BodyID"
+                },
+                "Latitude": {
+                    "type"          : "number",
+                    "description"   : "Must be added by the sender, generally from Status.json"
+                },
+                "Longitude": {
+                    "type"          : "number",
+                    "description"   : "Must be added by the sender, generally from Status.json"
+                }
```

Comment:
I think `Latitude` and `Latitude` need to be optional as they depend on status.json values from a live game.